python/src/FunctionalChaosSobolIndices.i

```diff
@@ -8,3 +8,107 @@
+    
+    # compute the mean, the variance, the standard deviation
+    randomVector = openturns.FunctionalChaosRandomVector(chaosResult)
+    mean = randomVector.getMean()
+    covarianceMatrix = randomVector.getCovariance()
+    stdDev = openturns.Point(outputDimension)
+    for i in range(outputDimension):
+        stdDev[i] = math.sqrt(covarianceMatrix[i, i])
+
+    # quick summary
+    basisSize = chaosResult.getReducedBasis().getSize()
+    html = ""
+    html += "<ul>\n"
+    html += "  <li>input dimension: %d</li>\n" % (inputDimension)
+    html += "  <li>output dimension: %d</li>\n" % (outputDimension)
+    html += "  <li>basis size: %d</li>\n" % (basisSize)
+    html += "  <li>mean: %s</li>\n" % (mean)
+    html += "  <li>std-dev: %s</li>\n" % (stdDev)
+    html += "</ul>\n"
+    
+    for i in range(outputDimension):
+        if i > maximumNumberOfOutput:
+            break
+        if outputDimension > 1:
+            html += "Output marginal: %d\n" % (i)
+
+        # Table of Sobol' indices
+        html += "<table>\n"
+        # Header
+        html += "  <tr>\n"
+        html += "    <th>Input</th>\n"
+        html += "    <th>Variable</th>\n"
+        html += "    <th>Sobol' index</th>\n"
+        html += "    <th>Total index</th>\n"
+        html += "  </tr>\n"
+        # Content
+        for j in range(inputDimension):
+            html += "  <tr>\n"
+            html += "    <td>%d</td>\n" % (j)
+            html += "    <td>%s</td>\n" % (inputDescription[j])
+            sobolIndex = self.getSobolIndex(j, i)
+            html += "    <td>" + sobolIndexFormat % sobolIndex + "</td>\n"
+            sobolIndex = self.getSobolTotalIndex(j, i)
+            html += "    <td>" + sobolIndexFormat % sobolIndex + "</td>\n"
+            html += "  </tr>\n"
+            
+        html += "</table>\n"
+
+        # Compute the part of variance of each multi-index
+        partOfVariance = self.getPartOfVariance(i)
+        # Sort marginal variances in decreasing order
+        marginalPartOfVarianceSample = openturns.Sample.BuildFromPoint(partOfVariance)
+        order = marginalPartOfVarianceSample.argsort(False)
+
+        # Table of part of variances indices
+        html += "<table>\n"
+        # Header
+        html += "  <tr>\n"
+        html += "    <th>Index</th>\n"
+        html += "    <th>Multi-index</th>\n"
+        html += "    <th>Part of variance</th>\n"
+        html += "  </tr>\n"
+        # Content
+        for k in range(basisSize):
+            sortedIndex = order[k]
+            globalIndex = indices[sortedIndex]
+            multiIndex = enumerateFunction(globalIndex)
+            partOfVarianceIndex = marginalPartOfVarianceSample[sortedIndex]
+            if partOfVarianceIndex[0] > partOfVarianceThreshold:
+                html += "  <tr>\n"
+                html += "    <td>%d</td>\n" % (sortedIndex)
```

Comment:
I would also prefer f-strings instead of old "%" formatting to write the html lines in a more readable manner

Reply:
Ok. Fixed at edc1bba1b